frontend/src/metabase/hooks/use-sequenced-content-close-handler.ts

```diff
@@ -90,5 +117,12 @@ export default function useSequencedContentCloseHandler() {
     [handleEvent, removeCloseHandler],
   );
 
+  useEffect(() => {
+    window.addEventListener("click", handleClickEvent, true);
+    return () => {
+      window.removeEventListener("click", handleClickEvent, true);
+    };
+  }, [handleClickEvent]);
```

Comment:
I always attach the click event because if I add and remove the event listener at the same place as the `mousedown` and `keydown` then when I hold click on the trigger when there's only popover, the popover will be closed, and the click listener will be lost. But I need the click listener to stay so I could track whether or not the click has been released. ![image](https://user-images.githubusercontent.com/1937582/161100066-bd0eb925-c314-4904-b21f-4fdf45f1dfce.png)